src/components/CreateResetPassword/CreateResetPasswordControl.js

```diff
@@ -38,11 +38,14 @@ class CreateResetPasswordControl extends Component {
     authFailure: [],
   };
 
+  state = {
+    isSuccessfulPasswordChange: false,
+  };
+
   handleSuccessfulResponse = () => {
-    // Todo: don't have back-end yet, should be implemented
+    this.setState({ isSuccessfulPasswordChange: true });
   };
 
-  // Todo: don't have back-end yet, should be changed
   handleSubmit = values => {
     const {
       mutator: { changePassword },
```

Comment:
Let's rewrite this method with async/await

Reply:
Cause the submit function is implemented in #513 PR, should it be fixed there?